lib/util/exec/buildpack.ts

```diff
@@ -93,7 +94,7 @@ const allToolConfig: Record<string, ToolConfig> = {
   python: {
     datasource: 'github-releases',
     depName: 'containerbase/python-prebuild',
-    versioning: pep440VersioningId,
+    versioning: poetryVersioningId,
```

Comment:
Sorry for the drive-by, was this meant to update all python versioning to poetry? This appears to break the following constraint in a Pipfile:

```
[requires]
python_version = "3.9"
```

As it's [parsed to](https://github.com/renovatebot/renovate/blob/b24d79662381764c7f51583c1feffe3632cb36b9/lib/modules/manager/pipenv/artifacts.ts#L34) `==3.9.*` which is part of PEP440.

Reply:
will fix that today

Reply:
- #17147